lib/plugins/Label_80.ts

```
@@ -37,145 +23,161 @@ export class Label_80 extends DecoderPlugin {
+    const kvPair = part.split(/\s+/)
+    if(kvPair.length < 2) {
+      ResultFormatter.unknown(results, part, '/');
+      return;
+    }
+    const tag = kvPair[0];
+    const val = kvPair.slice(1).join(' ');
+
+    switch (tag) {
+      case 'POS':
+      // don't use decodeStringCoordinates because of different position format
+        const posRegex = /^(?<latd>[NS])(?<lat>.+)(?<lngd>[EW])(?<lng>.+)/;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)